.github/workflows/check-header.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
     - name: Get changed files
       run: |
-        echo "files=$(git diff --name-only ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }} | grep -v '^obsolete' | tr '\n' ',')" >> $GITHUB_ENV
+        echo "files=$(git diff --name-only ${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }} | grep -v '^obsolete' | grep -v '^tests' | tr '\n' ',')" >> $GITHUB_ENV
```

Comment:
Any particular reason to exclude files in the test directory?

( I have no strong opinion here)

Reply:
Hm, there was a fail for the copyright header I did not quite get at first sight why that happens. Will revert and see what it was

Reply:
Yeah so if you check the most recent check run, it tries to validate that test vspecs do have copyright headers.
Do we want that? If yes, I need to add headers  to all  test files that I have touched